src/pivot-table/components/shared-styles.ts

```diff
@@ -73,6 +73,33 @@ export const getBorderStyle = (
   };
 };
 
+export const getHeaderBorderStyle = (
+  cell: HeaderCell,
+  isLastRow: boolean,
+  isFirstColumn: boolean,
+  isLastColumn: boolean,
+  borderColor: string,
+  showLastBorder?: ShowLastBorder,
+) => {
+  const headerBorderStyle = { ...getBorderStyle(isLastRow, isLastColumn, borderColor, showLastBorder) };
```

Comment:
Should be no need to spread the return value of `getBorderStyle`

```suggestion
  const headerBorderStyle = getBorderStyle(isLastRow, isLastColumn, borderColor, showLastBorder);
```